src/components/Hero.tsx

```diff
@@ -27,11 +27,13 @@ const HeroContent = styled.div`
 `;
 
 const smallItems = [
+	'Bättre mjukvara för en bättre värld',
```

Comment:
We should remove `smallItems` and `bigItems` and replace the constants with `items` since we are not using different texts/layouts anymore.